dev/diffs/4.0.0-preview1.diff

```diff
@@ -442,7 +442,7 @@ index 2c24cc7d570..50a2ce86117 100644
  
 -  test("partition pruning in broadcast hash joins with aliases") {
 +  test("partition pruning in broadcast hash joins with aliases",
-+    IgnoreComet("TODO: https://github.com/apache/datafusion-comet/issues/551")) {
```

Comment:
Hmm, aren't they already ignored? Seems this patch only changes the ignore reason?

Reply:
This 4.0.0 diff was created before #437 was merged, so these tests are not specifically related to Spark 4.0 issues that is tracked in #551

Reply:
I meant before this PR, the diffs already have `IgnoreComet` which ignores these tests. Isn't?

Reply:
Yes, that is correct, these tests are ignored currently, so no change in test coverage. The purpose of this PR is to tag them with #242 so that they will be fixed all together.

Reply:
Yea, thanks for confirming it. The PR title seems inaccurate so.

Reply:
Good point @parthchandra Would you like to update the title?

Reply:
Updated the commit and PR title.

Reply:
Thanks @parthchandra